pyomo/core/expr/numvalue.py

```diff
@@ -384,6 +413,7 @@ def as_numeric(obj):
         val = _KnownConstants.get(obj, None)
         if val is not None:
             return val 
+        print ".",
```

Comment:
@jsiirola I assume this print statement was supposed to be deleted?

Reply:
Yup.